pkg/gadgets/trace/bind/tracer/bpf/bindsnoop.h

```diff
@@ -32,4 +32,52 @@ union bind_options {
 	} fields;
 };
 
+
+/*
+ * Highly inspired from:
+ * https://github.com/iovisor/bcc/pull/4846/commits/a55b5a0c920f
+ */
+struct inet_sock___o {
+	__u8 freebind: 1;
+	__u8 transparent: 1;
+	__u8 bind_address_no_port: 1;
+};
+
+enum {
+	INET_FLAGS_FREEBIND___x = 11,
+	INET_FLAGS_TRANSPARENT___x = 15,
+	INET_FLAGS_BIND_ADDRESS_NO_PORT___x = 18,
+};
```

Comment:
I suppose we can't use `bpf_core_enum_value` with these ones as the enum is anonymous?